docs/troubleshooting.md

```diff
@@ -42,15 +42,37 @@ const {join} = require('path');
 /**
  * @type {import("puppeteer").Configuration}
  */
-module.exports = {
-  cacheDirectory: join(__dirname, '.cache', 'puppeteer'),
-};
+module.exports = {cacheDirectory: join(__dirname, '.cache', 'puppeteer')};
```

Comment:
Can we revert this style changes, they don't look good.
And to fix the linter we need to add `objectWrap: 'preserve',` to the prettier confgi